garnett/trajectory.py

```diff
@@ -494,66 +502,102 @@ def types(self, value):
         self.frame_data.types = value
 
     @property
-    def positions(self):
+    def position(self):
         "Nx3 array of coordinates for N particles in 3 dimensions."
         self.load()
-        return self._raise_attributeerror('positions')
+        return self._raise_attributeerror('position')
 
-    @positions.setter
-    def positions(self, value):
+    @property
+    def positions(self):
+        """
+        Nx3 array of coordinated for N particles in 3 dimensions.
+        Deprecated alias for position.
+        """
+        self._deprecation_warning('positions', 'position')
+        try:
+            return self.position(self)
+        except TypeError:
+            return self.position
+
+    @position.setter
+    def position(self, value):
         # Various sanity checks
         try:
             value = np.asarray(value, dtype=self._dtype)
         except ValueError:
-            raise ValueError("Positions can only be set to numeric arrays.")
+            raise ValueError("position can only be set to numeric arrays.")
         if not np.all(np.isfinite(value)):
-            raise ValueError("Positions being set must all be finite numbers.")
+            raise ValueError("position being set must all be finite numbers.")
         elif not len(value.shape) == 2 or value.shape[1] != self.box.dimensions:
             raise ValueError("Input array must be of shape (N,{}) where N is the "
```

Comment:
Is this right? I thought 2D systems still had 3D positions, just with z=0. I am pretty sure that is a convention we use across all group software.

Reply:
Yes I agree, this should be fixed. The question is whether are we throwing `ValueError` when `dimensions==3 and z != 0` or not.

Reply:
Is this for this PR as well? Or a seperate issue?

Reply:
@bdice what do you think about this?

Reply:
This has been fixed